packages/react-components/theme-designer/src/components/ColorTokens/ColorTokensList.tsx

```diff
@@ -0,0 +1,182 @@
+/* eslint-disable react/jsx-no-bind */
+import * as React from 'react';
+import { makeStyles } from '@griffel/react';
+import {
+  Divider,
+  Menu,
+  MenuButton,
+  MenuItemRadio,
+  MenuList,
+  MenuPopover,
+  MenuProps,
+  MenuTrigger,
+  tokens,
+  Subtitle2,
+} from '@fluentui/react-components';
+import type { AccentColors } from '../../utils/themes/createCustomLightTheme';
+import { Brands, BrandVariants } from '@fluentui/react-theme';
+import { CircleFilled } from '@fluentui/react-icons';
+
+export interface ColorTokensListProps {
+  brand: BrandVariants;
+  colors: AccentColors;
+  dispatchColors: React.Dispatch<{ colorToken: string; newValue: Brands }>;
+}
+
+const useStyles = makeStyles({
+  root: {},
+  colorLabel: {
+    color: tokens.colorBrandForeground1,
+  },
+  col: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'left',
+  },
+  row: {
+    paddingLeft: '5px',
+    paddingRight: '5px',
+    display: 'grid',
+    gridTemplateColumns: '1fr 1fr 1fr',
+    alignItems: 'center',
+    paddingTop: tokens.spacingVerticalXXXL,
+    paddingBottom: tokens.spacingVerticalXXXL,
+  },
+});
+
+export const ColorTokensList: React.FunctionComponent<ColorTokensListProps> = props => {
+  const styles = useStyles();
+  const { brand, colors, dispatchColors } = props;
+
+  const usageList = {
```

Comment:
break this out into its own file as well. hopefully we don't have to hard code this in the future